scripts/eval/eval.py

```diff
@@ -143,6 +156,10 @@ def evaluate_model(
         composer_model = load_model(model_cfg.model, tokenizer, fsdp_config,
                                     num_retries)
 
+    # Now add the eval metrics
+    if eval_loader_config is not None:
+        evaluators = add_metrics_to_eval_loaders(evaluators, composer_model)
```

Comment:
i think we can just pass `composer_model.train_metrics` in here instead of the whole model? Any particular reason you did it this way?

Reply:
also I think this should actually be `composer_model.get_metrics(is_train=True)`. thats the interface that is actually defined on `ComposerModel`. (this would be correcting an error from the code before your changes)